.stylelintignore

```diff
@@ -0,0 +1,2 @@
+src/css/vendor/**/*.css
```

Comment:
we are indirectly changing the rules by adding these files in `stylelintignore`

You're not supposed to have this file, kindly delete it.

Reply:
I am probably going to sound stupid, but by file you mean the src/css/vendor/**/*.css expression or the whole .stylelintignore file? Thank you.